packages/e2e/detox/integration/common/shared.ts

```diff
@@ -93,7 +93,18 @@ Then(
 );
 
 Then('I see placeholder {string}', async (placeholder: string) => {
-  await expect(element(by.text(placeholder))).toExist();
+  const emailField = element(
+    by.id('authenticator__text-field__input-username')
+  );
+  await expect(emailField).toExist();
+  const attributes = await emailField.getAttributes();
+  if ((attributes as Detox.ElementAttributes).placeholder) {
```

Comment:
Do we want to take any action if there is no `attributes.placeholder`?

Reply:
I think right now I'm not sure how we want to handle it because it seems like it isn't always `placeholder` based on the platform? I saw in some cases it could be `hint` or other options instead

Reply:
Ah, ok. So is the idea then that we're just covering those expected cases of `placeholder` here, and we can possibly expand to the other platforms in the future?

Reply:
The detox utils are specific to React Native